lib/_stream_readable.js

```diff
@@ -447,6 +447,7 @@ Readable.prototype._read = function(n) {
 Readable.prototype.pipe = function(dest, pipeOpts) {
   var src = this;
   var state = this._readableState;
+  pipeOpts = pipeOpts || {};
```

Comment:
should we care that in a typical use-case this would create an arbitrary object?